lib/utils.js

```diff
@@ -1,6 +1,18 @@
 import NativePromise from 'native-promise-only'
+import pkgJson from '../package.json'
+
 const Promise = window.Promise || NativePromise
 
+export function addUserAgent (headers, appId) {
```

Comment:
nit: maybe we can call it `updateUserAgentWithAppId` for its purpose? shall we worry the fallback value for appId if it is null

Reply:
Good idea, done in this commit: https://github.com/zendesk/zendesk_app_framework_sdk/pull/220/commits/23e6df883bfab63cfa495c7c8bed55be8b603c91